backend/src/eml/mod.rs

```diff
@@ -87,6 +87,13 @@ impl EML510 {
 
 impl base::EMLDocument for EML510 {}
 
+fn election_subcategory(election: &crate::election::Election) -> Option<String> {
+    match (&election.category, election.number_of_seats) {
+        (crate::election::ElectionCategory::Municipal, ..19) => Some("GR1".into()),
```

Comment:
I'm getting `error[E0658]: exclusive range pattern syntax is experimental` on my local `cargo run`. Our backend build jobs are green, though. So should I change a local setting or should our pipeline also fail on experimental features?

Reply:
This was actually stabilized in Rust 1.80, so I think you probably need to update your Rust version (i.e. `rustup update` if installed that way.

Reply:
Thanks! Was still on 1.78. :grimacing: